nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/ITListAzureBlobStorage.java

```diff
@@ -55,12 +55,58 @@ public void testListBlobsUsingCredentialService() throws Exception {
         assertResult();
     }
 
+    @Test
+    public void testListWithMinAge() throws Exception {
+        runner.setProperty(ListAzureBlobStorage.MIN_AGE, "1 hour");
+
+        runner.assertValid();
+        runner.run(1);
+
+        runner.assertTransferCount(ListAzureBlobStorage.REL_SUCCESS, 0);
+    }
+
+    @Test
+    public void testListWithMaxAge() throws Exception {
+        runner.setProperty(ListAzureBlobStorage.MAX_AGE, "1 hour");
+
+        runner.assertValid();
+        runner.run(1);
+
+        assertResult(TEST_FILE_CONTENT);
+    }
+
+    @Test
+    public void testListWithMinSize() throws Exception {
+        uploadTestBlob("nifi-test-blob2", "Test");
+        runner.setProperty(ListAzureBlobStorage.MIN_SIZE, "5 B");
+
+        runner.assertValid();
+        runner.run(1);
+
+        assertResult(TEST_FILE_CONTENT);
+    }
+
+    @Test
+    public void testListWithMaxSize() throws Exception {
+        uploadTestBlob("nifi-test-blob2", "Test");
+        runner.setProperty(ListAzureBlobStorage.MAX_SIZE, "5 B");
+
+        runner.assertValid();
+        runner.run(1);
+
+        assertResult("Test");
+    }
+
     private void assertResult() {
+        assertResult(TEST_FILE_CONTENT);
+    }
+
+    private void assertResult(final String contentLengthAsString) {
```

Comment:
The parameter is the actual content, not its length. Pls. correct the parameter name.